3696/1/presenter.rb

```diff
@@ -0,0 +1,22 @@
+require './scraper'
+require 'terminal-table'
+class Presenter
+  def initialize(hash, top)
+    @hash = hash
+    @top = top
+    @rows = []
+    generate_rows
+    table = Terminal::Table.new headings: ['Name', 'Watches', 'Stars', 'Forks', 'Contributors',
+                                           'Issues', 'Used by'], rows: @rows
+    puts table
+  end
+
+  def generate_rows
```

Comment:
call this method `rows` and pass into table instead of `@rows`
Do not use instance variable inside. You don't really need it